app/controllers/openid_connect/logout_controller.rb

```diff
@@ -5,7 +5,9 @@ class LogoutController < ApplicationController
     before_action :apply_secure_headers_override, only: [:index]
 
     def index
-      @logout_form = OpenidConnectLogoutForm.new(logout_params)
+      @logout_form = OpenidConnectLogoutForm.new(
+        logout_params.merge(current_user: current_user),
+      )
```

Comment:
probably clearer to update the constructor to take 2 args:
```suggestion
      @logout_form = OpenidConnectLogoutForm.new(
        params: logout_params,
        current_user: current_user,
      )
```